### PR TITLE
the changes in image_path

### PR DESCRIPTION
**Pull request by HarikrishnaBatta**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@ def destroy
     end
 
     def pending
-      users = User.where(approved: false)
+      users = User.where(approved: false).order(:emp_id).reverse
```
Comment (srihari-maddineni) [inline app/controllers/users_controller.rb]: @HarikrishnaBatta use asc, desc to change the order. not the reverse.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@ def destroy
...
       head :no_content if users.empty?
```
Comment (srihari-maddineni) [inline app/controllers/users_controller.rb]: this statement is not needed.
let the render json: return empty hash if not users are found.

File: app/models/device.rb [hunk 3]
```diff
@@ -20,11 +20,11 @@ def create_dev_no
     end
 
     def add_default_image
+     device_path = default_url
```
Comment (srihari-maddineni) [inline app/models/device.rb]: what is the use of this line. you can default_url, right?

Comment (HarikrishnaBatta) [reply app/models/device.rb]: to get the path and use in to conditions to generate appropriate urls

**Review by srihari-maddineni — APPROVED** [review]

**Pull request merged**